pages/quiz/[slug].tsx

```diff
@@ -68,7 +68,7 @@ function Quiz({ quizTitle, quizQuestions: originalQuizQuestions }: QuizProps) {
     });
     shuffleArray(randomizedQuestions);
     setQuizQuestions(randomizedQuestions);
-  }, []);
+  }, [originalQuizQuestions]);
```

Comment:
Was this line really needed?
If it's just for optimization / cleanup, it should be in a seperate PR.

Reply:
This was a warning so strictly speaking it didn't need pass the Vercel build but I was just fixing all the warnings that were obvious and seemed small enough.

Reply:
Hmm, I see. I'm not sure what the best approach is.

One one hand, it's good that we get these warnings fixed. 
On the other hand, I dislike the fact that it gets done in a PR dedicated to updating Next - unless the warnings came in after Next was updated.

Reply:
I can reintroduce the error by reverting some commits and someone can create the PR.

Reply:
Hmm, that _might_ be the right approach.

It's not that I don't like the fix - I just want a "seperation", if that makes sense. A PR should be dedicated to a specific issue, without making other fixes along the way, unless they are related 👍

Reply:
Re-introduced... 1a145a0

![image](https://user-images.githubusercontent.com/75389/134788756-16ab45ee-51d4-4b71-a11e-72a82b2978cb.png)

Reply:
Great. I don't like warnings either, but if it's not strictly related to Next11, it should be fixed in another issue 👍 .. I'll create an issue for it :)